soap-attachments/2.0/apidocs/jquery/jquery-ui.css

```diff
@@ -1,8 +1,8 @@
-/*! jQuery UI - v1.12.1 - 2018-12-06
+/*! jQuery UI - v1.11.4 - 2015-05-20
```

Comment:
Was there a reason we dropped to a lower version?

Reply:
Different JDK on VMs